WordPress/gutenberg
# [RNMobile] Update Inserter Block Search Styling

## Description
This updates the inserter block search styling to handle variances between iOS and Android. It also introduces a no results view.


## How has this been tested?
1. Load the demo app in both IOS and Android
2. Open the Inserter
3. Verify that the styling looks correct in light and dark mode
4. Press on the search input
5. Verify that the styling for active search looks correct.
6. Try entering any text.
7. Verify that the cancel input options look correct and clear out the text input when pressed
8. Enter more text and press on the cancel search options.
9. Verify that the bottom sheet is closed without inserting a block
10. Open the Inserter and enter arbitrary text that does not match to a block e.g. `asdf`
11.  Verify that the "No block found" message is displayed.
12. Press backspace on the inputed text 
13. Verify that possible query results are rendered in the inserter e.g. removing characters from `asdf` until only `as` remains.

## Screenshots 
### Open Inserter

Android|iOS
--|--
![Screenshot_20210708-155752](https://user-images.githubusercontent.com/744755/124988498-0ef37e00-e00c-11eb-824c-6560f6af7204.png)|![IMG_0402](https://user-images.githubusercontent.com/744755/124988522-16b32280-e00c-11eb-9f1a-8b40e6d1a78d.PNG)
![Screenshot_20210708-155916](https://user-images.githubusercontent.com/744755/124988685-49f5b180-e00c-11eb-839d-123b774a1efe.png)|![IMG_0407](https://user-images.githubusercontent.com/744755/124988624-39ddd200-e00c-11eb-8a77-8fd410bfa635.PNG)

### Start Search
Android|iOS
--|--
![Screenshot_20210708-162619](https://user-images.githubusercontent.com/744755/124989853-98578000-e00d-11eb-8948-93bcb771bb23.png)|![IMG_0403](https://user-images.githubusercontent.com/744755/124989883-a0afbb00-e00d-11eb-9691-6dea981f0e34.PNG)
![Screenshot_20210708-165018](https://user-images.githubusercontent.com/744755/124989918-ad341380-e00d-11eb-9691-046798285895.png)|![IMG_0405](https://user-images.githubusercontent.com/744755/124989953-ba510280-e00d-11eb-91a8-3211b9b323cb.PNG)


### No results

Android|iOS
--|--
![Screenshot_20210708-162641](https://user-images.githubusercontent.com/744755/124990055-db195800-e00d-11eb-8ca7-2f144d9650a1.png)|![IMG_0404](https://user-images.githubusercontent.com/744755/124990093-ea98a100-e00d-11eb-8e9f-2bf609cc5e8d.PNG)
![Screenshot_20210708-162532](https://user-images.githubusercontent.com/744755/124990066-e10f3900-e00d-11eb-95cc-4cb0aecfa582.png)|![IMG_0406](https://user-images.githubusercontent.com/744755/124990119-f1bfaf00-e00d-11eb-8fc1-77e9765eb961.PNG)

## Types of changes




This adds new mobile platform specific style sheets and a new native component to show the no results view.

## Checklist:
